gui/pregnancy.lua

```diff
@@ -3,18 +3,32 @@ local widgets = require('gui.widgets')
 
 PregnancyGui = defclass(PregnancyGui, widgets.Window)
 PregnancyGui.ATTRS {
-    frame_title='My Window',
-    frame={w=50, h=45},
+    frame_title='Pregnancy manager',
+    frame={w=64, h=35},
     resizable=true, -- if resizing makes sense for your dialog
     resize_min={w=50, h=20}, -- try to allow users to shrink your windows
 }
 
 function PregnancyGui:init()
-    self.mother = false
+    if dfhack.gui.getSelectedUnit(true).sex == df.pronoun_type.she then
+        self.mother = dfhack.gui.getSelectedUnit(true)
+    else self.mother = false
```

Comment:
unless a field is always intended to be a boolean, it's better to leave unset values unset so they are `nil` instead of `false`.